lib/rmt/config.rb

```diff
@@ -1,3 +1,4 @@
+# :nocov:
```

Comment:
Same comment here, it should be enough to either skip coverage via `add_filter('lib/rmt/config.rb')` or the `:nocov:` lines.